pkg/planner/core/planbuilder.go

```diff
@@ -4389,6 +4389,10 @@ func (b *PlanBuilder) buildImportInto(ctx context.Context, ld *ast.ImportIntoStm
 	}
 
 	tnW := b.resolveCtx.GetTableName(ld.Table)
+	if tnW.TableInfo.TempTableType != model.TempTableNone ||
+		tnW.TableInfo.TableCacheStatusType != model.TableCacheStatusDisable {
+		return nil, errors.Errorf("IMPORT INTO does not support temporary table or cached table")
```

Comment:
Could you separate this message into "IMPORT INTO does not support temporary table" and "IMPORT INTO does not support cached table"? It is more clear and easy to extend to more checks if needed in the future